pkgs/top-level/all-packages.nix

```diff
@@ -1493,7 +1493,24 @@ with pkgs;
 
   pcsxr = callPackage ../applications/emulators/pcsxr { };
 
-  ppsspp = libsForQt5.callPackage ../applications/emulators/ppsspp { };
+  ppsspp-qt = libsForQt5.callPackage ../applications/emulators/ppsspp {
+    pname = "ppsspp-qt";
+    enableVulkan = false; # https://github.com/hrydgard/ppsspp/issues/11628
+  };
+
+  ppsspp-sdl = callPackage ../applications/emulators/ppsspp {
```

Comment:
After the rebase, write this and the other expressions in the form `ppsspp-sdl = ppsspp.override { . . . };`.